pkgs/tools/package-management/appimage-run/default.nix

```diff
@@ -14,7 +20,7 @@ let
     mimeTypes = ["application/vnd.appimage" "application/x-iso9660-appimage"];
     categories = ["PackageManager" "Utility"];
   };
-in buildFHSEnv (fhsArgs // {
+in buildFHSEnv (lib.recursiveUpdate fhsArgs {
```

Comment:
AFAICT, meta is not set in the default args.

Reply:
Just in case it is set in the future, we don't want to accidentally overwrite it here, so we are avoiding potential future errors.